File: five-languages/vowel-count/vowel_count.cs
// vowel_count.cs - Tæl antal vokaler i en streng (C#)

using System;

class Program {
    static bool IsVowel(char c) {
        return "aeiouAEIOU".IndexOf(c) >= 0;
    }

    static void Main() {
        Console.Write("Indtast en streng: ");
        string input = Console.ReadLine();
        int count = 0;

        foreach (char c in input) {
            if (IsVowel(c)) count++;
        }

        Console.WriteLine("Antal vokaler: " + count);
    }
}

File: five-languages/vowel-count/vowel_count.py
# vowel_count.py
# Tæl antal vokaler i en streng (Python)

def count_vowels(s):
    vowels = "aeiouAEIOU"
    return sum(1 for char in s if char in vowels)

if __name__ == "__main__":
    user_input = input("Indtast en streng: ")
    count = count_vowels(user_input)
    print("Antal vokaler:", count)
